ddtrace/tracer/tracer.go

```diff
@@ -228,6 +239,15 @@ func newUnstartedTracer(opts ...StartOption) *tracer {
 	if spans != nil {
 		c.spanRules = spans
 	}
+	var dataStreamsProcessor dsminterface.Processor
+	if c.enableDataStreamsMonitoring {
+		if c.agent.DataStreams {
+			log.Info("Enable Data Streams Monitoring")
```

Comment:
Let's remove this line. We have data streams support noted in the tracer startup log and don't want to clutter user logs.